<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HAngleTestSetWidget</class>
 <widget class="QWidget" name="HAngleTestSetWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_009">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>电流档位：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboBox_2</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_003">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>测量模式：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboBox_1</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="comboBox_1"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_002">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>输出电压：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBox_2</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_010">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>光档位：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>comboBox_3</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox_3"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_008">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>测量角度：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>spinBox_1</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBox_1"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_004">
     <property name="minimumSize">
      <size>
       <width>72</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>输出电流：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>doubleSpinBox_3</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
